README.md

```diff
@@ -162,6 +162,14 @@ You can skip these and only define your own units by doing:
 gem 'measured', require: 'measured/base'
 ```
 
+### Shortcut syntax
```

Comment:
Should we  update the README to use the nice syntax in all the examples, so it's the one proper way to do it?
> Initialize a measurement:

```ruby
Measured::Weight.new("12", "g")
```